workers/data_refinery_workers/processors/test_no_op.py

```diff
@@ -24,181 +83,127 @@ def setUp(self):
-        job.pipeline_applied = "NO_OP"
-        job.save()
-
-        # ex:
-        # ILMN_1885639    10.0000 0.7931
-        # ILMN_2209417    10.0000 0.2029
-        # ILMN_1765401    152.0873    0.0000
-        og_file = OriginalFile()
-        og_file.source_filename = (
-            "https://github.com/AlexsLemonade/refinebio/files/2255178/GSM1089291-tbl-1.txt"
+        """Illumina file without header, ex:
+        > ILMN_1885639    10.0000   0.7931
+        > ILMN_2209417    10.0000   0.2029
+        > ILMN_1765401    152.0873  0.0000
+        """
+        job = prepare_job(
+            {
+                "accession_code": "GSM1089291",
+                "source_filename": "https://github.com/AlexsLemonade/refinebio/files/2255178/GSM1089291-tbl-1.txt",
+                "filename": "GSM1089291-tbl-1.txt",
+                "absolute_file_path": "/home/user/data_store/raw/TEST/NO_OP/GSM1089291-tbl-1.txt",
+                "platform_accession_code": "A-MEXP-1171",
+                "manufacturer": "ILLUMINA",
+            }
         )
-        og_file.filename = "GSM1089291-tbl-1.txt"
-        og_file.absolute_file_path = "/home/user/data_store/raw/TEST/NO_OP/GSM1089291-tbl-1.txt"
-        og_file.is_downloaded = True
-        og_file.save()
+        final_context = assertRunsSuccessfully(self, job)
 
-        homo_sapiens = Organism(name="HOMO_SAPIENS", taxonomy_id=9606, is_scientific_name=True)
-        homo_sapiens.save()
-
-        sample = Sample()
-        sample.accession_code = "GSM557500"
-        sample.title = "GSM557500"
-        sample.platform_accession_code = "A-MEXP-1171"
-        sample.manufacturer = "ILLUMINA"
-        sample.organism = homo_sapiens
-        sample.save()
-
-        assoc = OriginalFileSampleAssociation()
-        assoc.original_file = og_file
-        assoc.sample = sample
-        assoc.save()
-
-        assoc1 = ProcessorJobOriginalFileAssociation()
-        assoc1.original_file = og_file
-        assoc1.processor_job = job
-        assoc1.save()
+        self.assertTrue(no_op.check_output_quality(final_context["output_file_path"]))
 
         # To:
         # ENSG00000105675 10
         # ENSG00000085721 152.0873
         # ENSG00000278494 152.0873
-        final_context = no_op.no_op_processor(job.pk)
-        self.assertTrue(final_context["success"])
-        self.assertTrue(os.path.exists(final_context["output_file_path"]))
-        self.assertTrue(
-            math.isclose(os.path.getsize(final_context["output_file_path"]), 789756, rel_tol=0.05)
-        )
+        expected_data = pd.read_csv(
+            "/home/user/data_store/TEST/NO_OP/EXPECTED/gene_converted_GSM1089291-tbl-1.txt",
+            sep="\t",
+            names=["", "VALUE"],
+            index_col=0,
+        )["VALUE"]
+        actual_data = pd.read_csv(
+            final_context["output_file_path"], sep="\t", names=["", "VALUE"], index_col=0
+        )["VALUE"]
+
+        assertMostlyAgrees(self, expected_data, actual_data)
```

Comment:
I like how succinct all these tests are now that you've refactored all the setup away :+1: